### PR TITLE
Remove deprecated methods (2019-05)

### PR DESCRIPTION
**Pull request by jwnimmer-tri**
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/11358)

### CODE REVIEW

**Comment by jwnimmer-tri** [comment]
+@sammy-tri to kick off both reviews, please.

For anything you don't want to be both reviewers on, please post a defect on those files to get a second reviewer in the picture.

File: bindings/pydrake/multibody/all.py [hunk 1]
```diff
@@ -3,14 +3,6 @@
 # Deprecated symbols.
 with warnings.catch_warnings():
     warnings.simplefilter("ignore", DeprecationWarning)
-    # - Attic (#9314)
-    from .collision import *
-    from .joints import *
-    from .parsers import *
-    from .rigid_body_plant import *
-    from .rigid_body_tree import *
-    from .rigid_body import *
```
Comment (RussTedrake) [inline bindings/pydrake/multibody/all.py]: Note: this change torpedoed my underactuated notes (there was no warning on that side suggesting these would go away)

https://circleci.com/gh/RussTedrake/underactuated/1193?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification

File: bindings/pydrake/multibody/BUILD.bazel [hunk 2]
```diff
@@ -136,23 +132,6 @@ drake_pybind_library(
     ],
 )
 
-PY_LIBRARIES_ATTIC_ALIASES = [
-    ":collision_py",
-    ":joints_py",
-    ":parsers_py",
-    ":rigid_body_plant_py",
-    ":rigid_body_py",
-    ":rigid_body_tree_py",
```
Comment (RussTedrake) [inline bindings/pydrake/multibody/BUILD.bazel]: or it might have actually been this one.

**Comment by jwnimmer-tri** [comment]
@EricCousineau-TRI can you investigate why the deprecation warnings did not occur in `underactuated`?  It is because `from pydrake.all import ...` muted the warning?  Or was my PR in error, that I should have respelled the `all` imports instead of removing them?

**Comment by jwnimmer-tri** [comment]
My guess:
(1) Using `from all import (..., RigidBodyPlant, ...)` worked before, and still works.
(2) Using `from pydrake.multibody.rigid_body_tree import (..., RigidBodyPlant, ...)` generated a deprecation warning.
(3) Mixing and matching (1) first and then (2) second in the same program did _not_ generate a deprecation warning.

I guess the question is, do we want to put back the deprecation forwarders for another month, but with the (3) caveat solved, so that other users are not also surprised?

**Pull request merged**